crates/sui-framework/tests/math_tests.move

```diff
@@ -18,6 +18,20 @@ module sui::math_tests {
         assert!(math::min(100, 10) == 10, 2);
         assert!(math::min(0, 0) == 0, 3);
     }
+    
+    #[test]
+    fun test_pow() {
+        assert!(nft::pow(1, 0) == 1, 0);
+        assert!(nft::pow(1, 1) == 1, 0);
```

Comment:
Nit: suggest using a different `base` than `1` to get a different value for this call vs the one above.
```suggestion
        assert!(nft::pow(3, 1) == 3, 0);
```